src/grib2_int.h

```diff
@@ -27,6 +27,9 @@
 
 #include "grib2.h"
 
+/** Define is system is Arm Linux. */
+#define IS_ARM_LINUX __aarch64__ &&__linux__
```

Comment:
We don't want this define.

Is there a way to detect whatever is happening on ARM and handle it without having platform specific defines?